disbursement/event_test.go

```diff
@@ -0,0 +1,83 @@
+package disbursement_test
+
+import (
+	"encoding/json"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/xendit/xendit-go/disbursement"
+)
+
+func TestPayload_Completed(t *testing.T) {
+	payload := `{
```

Comment:
📝 copypasted this payload from the docs 
https://developers.xendit.co/api-reference/#disbursement-callback